src/ast/ast_validations.c

```diff
@@ -135,18 +135,18 @@ static AST_Validation _AST_ValidateReferredFunctions(TrieMap *referred_functions
     return res;
 }
 
-static AST_Validation _MATCH_Clause_Validate_Range(const cypher_astnode_t *node, char **reason) {
+static AST_Validation _MATCH_Clause_Validate_Range(const cypher_astnode_t *range, char **reason) {
```

Comment:
So many validations...

Reply:
Ick, I know. We'll be able to remove some of them over time, at least, but to some degree there's no way around it unless we want to write them into the parser.